--- config.mk.orig	2025-05-16 03:06:59.667980589 -0400
+++ config.mk	2025-05-16 03:07:18.920309727 -0400
@@ -8,8 +8,8 @@
 MANDIR = $(PREFIX)/share/man
 DATADIR = $(PREFIX)/share
 
-WLR_INCS = `$(PKG_CONFIG) --cflags wlroots-0.19`
-WLR_LIBS = `$(PKG_CONFIG) --libs wlroots-0.19`
+WLR_INCS = `$(PKG_CONFIG) --cflags wlroots-0.20`
+WLR_LIBS = `$(PKG_CONFIG) --libs wlroots-0.20`
 
 # Allow using an alternative wlroots installations
 # This has to have all the includes required by wlroots, e.g:
